apps/react-18-tests-v9/src/components/Provider/Provider.cy.tsx

```diff
@@ -1,17 +1,18 @@
 import * as React from 'react';
 import { mount } from '@cypress/react';
```

Comment:
I might be missing something, but are we actually testing with cypress that supports React 18? 

We have in the root `cypress/react@8.0.0` and `cypress@13.6.4` inside as well has the one that seem to support <= 17. 

From migration docs: 
![image](https://github.com/user-attachments/assets/7f81c578-8c1f-4590-9081-e69845bd0ac1)

Reply:
This doesn't add any support for React 18 yet. This just reorganize the files and folders so we can add the tests correctly later.

Reply:
Gotcha, thanks ;)